charts/lagoon-core/templates/api.ingress.yaml

```diff
@@ -7,11 +7,17 @@ metadata:
   name: {{ $fullName }}
   labels:
     {{- include "lagoon-core.api.labels" . | nindent 4 }}
-  {{- with .Values.api.ingress.annotations }}
   annotations:
```

Comment:
I think it would be nice to avoid having this empty by wrapping it in a conditional, or at least enabling an ingress in CI so it is tested in all the k8s versions. Not a showstopper however.